tests/test_routes.py

```diff
@@ -50,6 +50,18 @@ def tearDown(self):
         db.session.remove()
 
 
+    def _create_product_id(self, count):
+            """Factory method to create pets in bulk"""
```

Comment:
Small nitpick, change pets to products. Can be addressed in next sprint

Reply:
fixed